data_structures/src/chain.rs

```diff
@@ -904,6 +910,8 @@ pub struct DataRequestOutput {
     // >50 because simple majority
     // <100 because a 100% consensus encourages to commit a RadError for free
     pub min_consensus_percentage: u32,
+    // This field must be > collateral_minimum
```

Comment:
Do we allow collaterals with exactly the minimum value?

```suggestion
    // This field must be >= collateral_minimum
```